src/components/Participant/RegistrationProcess/StartRegistrationProcess/StartRegistrationProcess.tsx

```diff
@@ -2,13 +2,45 @@ import DocumentTitle from "react-document-title";
 import StepWrapper from "../../../Shared/StepWrapper/StepWrapper";
 import { ContentContext } from "../../../../context/ContentContext";
 import { useContext } from "react";
+import DTEButton from "../../../Shared/UI/DTEButton/DTEButton";
+import { useHistory } from "react-router-dom";
+import NhsLoginButton from "../../../Shared/UI/NhsLoginButton";
+import DTEContent from "../../../Shared/UI/DTETypography/DTEContent/DTEContent";
+import DTELinkButton from "../../../Shared/UI/DTELinkButton/DTELinkButton";
 
 function StartRegistrationProcess() {
   const { content } = useContext(ContentContext);
+  const history = useHistory();
+
+  const handleGoBack = () => {
+    history.goBack();
+  };
 
   return (
     <DocumentTitle title={content["register-page-document-title"]}>
-      <StepWrapper>{content["register-page"]}</StepWrapper>
+      <StepWrapper>
+        <a href="#" onClick={handleGoBack}>
```

Comment:
Have a look at the `<DTEBackLink />` component that is already available in the code.